pyproject.toml

```diff
@@ -28,8 +28,8 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
-"discord.py" = "^1.7.3"
+python = "^3.10"

```

Comment:
Why python 3.10 is required ? As I see discord.py still support python 3.8

Reply:
Not justified, reverted in commit [15e67d3](https://github.com/lyon-esport/LouisDeLaTech/pull/185/commits/15e67d37bbbf3798c095c9719b9f8be8f54bb61e)